sdcm/cluster.py

```diff
@@ -3356,6 +3361,14 @@ def check_nodes_running_nemesis_count(self):
         ClusterHealthValidatorEvent(type='warning', name='NodesNemesis', status=Severity.WARNING,
                                     message=f"There are more then expected nodes running nemesis: {message}")
 
+    @retrying(n=30, sleep_time=10, allowed_exceptions=(AssertionError,))
```

Comment:
Seems to long to me, what is the maximum time we should get an agreement in Scylla?

Reply:
I let 5 minutes for be sure. I don't have a statistic for that. It depend on cluster size, schema, load..